### DISCUSSION TITLE
Should 'java.maven.downloadSources' be enabled by default?

### DISCUSSION THREAD

**Discussion by rahon6000** [redhat-developer/vscode-java#3361] [General]
Hello, 

Maybe I'm asking this because I'm quite new to Java with VScode.

Is there any fair reason for `java.maven.downloadSources` to be disabled by default?

With this setting disabled, every javadoc is missing in the project and I think it is quite frustrating for new comer.

Please let me know your opinion. 😁

**Comment by fbricon** [comment]
By default, Javadoc should be lazily downloaded on hover, when missing. If it doesn't, it's a bug.

**Reply by rahon6000** [reply]
I see... I will try to reproduce it soon. Thank you for the comment!
